src/opendap/bes/Version.java

```diff
@@ -43,8 +43,8 @@
  * Contains the Version and UUID information for Hyrax Server.
  */
 public class Version  {
-    private static final String olfsVersion  = "@OlfsVersion@";
-    private static final String hyraxVersion = "@HyraxVersion@";
+    private static final String olfsVersion  = "Not.A.Release";
```

Comment:
This should be handle by code filtering at build time. I don't think we want the version to be hardwired to "Not.A.Release"